### ISSUE TITLE
Unexpected behavior when a heading in Russian is present

### ISSUE THREAD

**Issue by fedorov** [billryan/gitbook-plugin-etoc#4]
To reproduce, add a section with a title in Russian. The result is that the section appears before the etoc-generated TOC.

Example:
```
#
еуые

Еуые

# Frequently Asked Questions (FAQ)

## Why?
```

Resulting rendering:
![image](https://cloud.githubusercontent.com/assets/313942/25248597/d0b2f74e-25dc-11e7-9337-b0d43fc87c42.png)

**Comment by billryan** [comment]
Currently the etoc would insert TOC before the first `##` Header 2 section, so it is a bit weird for your case. I will consider insert TOC after the first `#` Header 1 section.

**Comment by fedorov** [comment]
I don't think this is the reason. If i just change the language to English, everything works as expected and TOC shows up at the very top.

**Comment by billryan** [comment]
Seems this is the same bug with #2 ? If you do not have `###` header3.
Anyway, the default location for TOC should be improved, I am working on it.

**Comment by billryan** [comment]
This unexpected behaviour should be fixed in 1.1.10, and after 1.2.0, the TOC would show after the H1 header by default, configured by `header`.

P.S. The example(heading title and sharp symbol not in the same line) you gave
```
#
еуые
```
would break some format plugin with markdown, I recommend write the header title and sharp symbol(#) on the same line with space separated like
```
# еуые
```

**Comment by fedorov** [comment]
Thank you! I confirm the issue has been resolved.

> The example(heading title and sharp symbol not in the same line) you gave would break some format plugin with markdown

That was not intentional!
